### ISSUE TITLE
"transpose" in voice ignored

### ISSUE THREAD

**Issue by dvdrndlph** [paulrosen/abcjs#33]
From the ABC v2.1 standard, we have this example. The transpose instructions in the bass clef voices are being ignored in ABCJS. 

Also, the fermatas in the final measure should be combined and a beam seems to be detached in the second to last measure of the bass clef line.

I see these behaviors in the 2.0 version of ABCJS.

``` X:1
T:Zocharti Loch
C:Louis Lewandowski (1821-1894)
M:C
Q:1/4=76
%%score (T1 T2) (B1 B2)
V:T1           clef=treble-8  name="Tenore I"   snm="T.I"
V:T2           clef=treble-8  name="Tenore II"  snm="T.II"
V:B1  middle=d clef=bass      name="Basso I"    snm="B.I"  transpose=-24
V:B2  middle=d clef=bass      name="Basso II"   snm="B.II" transpose=-24
K:Gm
%            End of header, start of tune body:
% 1
[V:T1]  (B2c2 d2g2)  | f6e2      | (d2c2 d2)e2 | d4 c2z2 |
[V:T2]  (G2A2 B2e2)  | d6c2      | (B2A2 B2)c2 | B4 A2z2 |
[V:B1]       z8      | z2f2 g2a2 | b2z2 z2 e2  | f4 f2z2 |
[V:B2]       x8      |     x8    |      x8     |    x8   |
% 5
[V:T1]  (B2c2 d2g2)  | f8        | d3c (d2fe)  | H d6    ||
[V:T2]       z8      |     z8    | B3A (B2c2)  | H A6    ||
[V:B1]  (d2f2 b2e'2) | d'8       | g3g  g4     | H^f6    ||
[V:B2]       x8      | z2B2 c2d2 | e3e (d2c2)  | H d6    ||
```

**Comment by paulrosen** [comment]
I haven't implemented transpose yet. I'll bump it up on the list. 

I think your example got eaten by the markup. Can you post it as code?

**Comment by paulweichhartgugler** [comment]
Any news regarding the transpose functionality? Would love to see this feature implemented...
Thanks in advance!

**Comment by rpbouman** [comment]
I'm using abcjs 3.1.4 and I'm running into the same issue.

**Comment by paulrosen** [comment]
Will be in the upcoming v4.0.0 release.

**Comment by paulrosen** [comment]
Fixed in 4.0.0

**Comment by ChorHizzle** [comment]
I just ran into the issue when trying it out in the editor (I'm guessing the editor hasn't been updated to 4.0.0):
https://abcjs.net/abcjs-editor.html
